From 51b53da348696dbad07aad97670f8096333cff83 Mon Sep 17 00:00:00 2001
From: Antoine Martin <dev@ayakael.net>
Date: Sat, 12 Feb 2022 23:32:32 +0000
Subject: [PATCH 1/2] build_copy-sbrp

By default, build moves SourceBuildReferencePackages subfolder its own
folder. Given that artifacts are in a read-only location under this
aport, this patch makes build copy instead of move the subfolder

---
 tools-local/init-build.proj | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/tools-local/init-build.proj b/tools-local/init-build.proj
index c5494dedd..1070cf0be 100644
--- a/tools-local/init-build.proj
+++ b/tools-local/init-build.proj
@@ -57,13 +57,13 @@
           WorkingDirectory="$(PrebuiltPackagesPath)"
           Condition="'@(SourceBuiltPrebuiltsTarballFile)' != ''" />
 
-    <!-- Move SBRP packages to reference packages location -->
+    <!-- Copy SBRP packages to reference packages location -->
     <MakeDir Directories="$(ReferencePackagesDir)" />
     <ItemGroup>
       <UnpackedSourceBuildReferencePackages Include="$(PrebuiltSourceBuiltPackagesPath)SourceBuildReferencePackages/*"/>
     </ItemGroup>
 
-    <Move SourceFiles="@(UnpackedSourceBuildReferencePackages)" DestinationFiles="$(ReferencePackagesDir)%(Filename)%(Extension)" />
+    <Copy SourceFiles="@(UnpackedSourceBuildReferencePackages)" DestinationFiles="$(ReferencePackagesDir)%(Filename)%(Extension)" />
 
     <!-- remove some reference packages that are generated incorrectly and instead use the prebuilt checked-in version instead -->
     <!-- relevant issues: https://github.com/dotnet/runtime/issues/44646, https://github.com/dotnet/runtime/issues/45183,
-- 
2.34.1

